### PR TITLE
initialize, open and close metrics client

### PR DESCRIPTION
**Pull request by avsek477**
Depends on: 
- https://github.com/bitfinexcom/bfx-hf-algo/pull/210
- https://github.com/bitfinexcom/bfx-hf-metrics-server/pull/2
- https://github.com/bitfinexcom/bfx-hf-strategy/pull/26
- https://github.com/bitfinexcom/bfx-hf-strategy-exec/pull/24

Metrics server integration.

Returns the following response upon successful authentication.
`['info.auid', auid, {mode: 'main', scope: 'app'}]`

### CODE REVIEW

File: package.json
Comment (avsek477) [inline package.json:25]: needs to be changed after merge: https://github.com/bitfinexcom/bfx-hf-algo/pull/210

File: package.json
Comment (avsek477) [inline package.json:30]: wait for merge: https://github.com/bitfinexcom/bfx-hf-strategy/pull/26

File: package.json
Comment (avsek477) [inline package.json:31]: wait for merge: https://github.com/bitfinexcom/bfx-hf-strategy-exec/pull/24

File: lib/ws_servers/api/session.js
Comment (tarcisiozf) [inline lib/ws_servers/api/session.js:114]: docs doesn't match

File: lib/ws_servers/api/session.js
Comment (tarcisiozf) [inline lib/ws_servers/api/session.js:134]: this `data` args overrides the one above.

File: lib/ws_servers/api/start_connections.js
Comment (tarcisiozf) [inline lib/ws_servers/api/start_connections.js:134]: await is not required

File: lib/ws_servers/api/metrics_client.js
Comment (tarcisiozf) [inline lib/ws_servers/api/metrics_client.js:12]: I rather throw an exception than use a default localhost URL

**Review by tarcisiozf — CHANGES_REQUESTED** [review]
looks good, left some comments

File: lib/ws_servers/api/session.js
Comment (avsek477) [reply lib/ws_servers/api/session.js:134]: the data arg above is not required.. forgot to remove it later. thanks!!

**Review by tarcisiozf — APPROVED** [review]

File: lib/ws_servers/api/session.js
Comment (avsek477) [inline lib/ws_servers/api/session.js:123]: also, need to add a condition in `closeServices` method to close the metrics client if it's available and open.
`if(this.metricsClient && this.metricsClient.isOpen) this.metricsClient.close()`

Comment (tarcisiozf) [reply lib/ws_servers/api/session.js:123]: thanks for catching that

**Pull request merged**
